erpnext/accounts/utils.py

```diff
@@ -112,17 +115,37 @@ def get_balance_on(account=None, date=None, party_type=None, party=None, company
 			# hence, assuming balance as 0.0
 			return 0.0
 
+	allow_cost_center_in_entry_of_bs_account = frappe.db.get_value('Accounts Settings', None, \
```

Comment:
move this to a function?

Reply:
Thank you @rmehta for the review. I will make required changes.